src/test/java/com/onfido/integration/TestBase.java

```diff
@@ -188,12 +189,12 @@ public Object repeatRequestUntilStatusChanges(
       Thread.sleep(sleepTime);
 
       if (isInstanceOfReport) {
-        instance = ((Report) method.invoke(onfido, params)).getActualInstance();
+        instance = ((Report) method.invoke(onfido, params)).getReportShared();
```

Comment:
Actually this should be enough:
```suggestion
        instance = ((Report) method.invoke(onfido, params));
```